skypy/galaxies/stellar_mass.py

```diff
@@ -74,3 +78,131 @@ def schechter_smf_mass(redshift, alpha, m_star, m_min, m_max, size=None,
     m = schechter(alpha, x_min, x_max, resolution, size=size, scale=m_star)
 
     return m
+
+
+def schechter_smf_amplitude_centrals(phi_blue_total, fsatellite):
+    r'''Schechter amplitude of centrals.
+    This function returns the Schechter mass function amplitude
+    for active central population based on equation (15)
+    in de la Bella et al. 2021 [1]_.
+
+    Parameters
+    ----------
+    phi_blue_total: float, (nt, ) array_like
+        Schechter mass function amplitude for the entire active
+        sample of galaxies, :math:`(\phi_b, \alpha_b, m_{*})`.
+
+    fsatellite: float, (nm, ) array_like
+        Fraction of active satellite galaxies between 0 and 1.
+        It could be a float or an array, depending on the model you choose.
+
+    Returns
+    -------
+    amplitude: float, (nt, nm) array_like
+        Amplitude of the Schechter mass function.
```

Comment:
@rrjbca  It might be the case where phi will have a time and a mass dependence. `phi_blue_total` could be a scalar but it can also be an array of size `nt`. Likewise, `fsatellite` can be a scalar or an array of size `nm`.  If `nt != nm` the product cannot be broadcasted.

Reply:
I don't see any reason not to make the function more flexible and allow it to take any two arrays with broadcastable shapes. If you have two 1D arrays and want the result for every combination the way to broadcast them would be:
```python
>>> x = np.arange(5)
>>> y = np.arange(7)
>>> result = schechter_smf_amplitude_centrals(x, y[:, np.newaxis])
>>> result.shape
(5, 7)
```